docs/source/host-collect-analyze/cpu.md

```diff
@@ -42,7 +42,21 @@ The `cpu` analyzer supports multiple outcomes by validating the number of CPU co
 - `count < 32`: Less than 32 CPU cores were detected.
 - `count > 4`: More than 4 CPU cores were detected.
 
-### Example Analyzer Definition
+The analyzer also supports validating the presence of specific CPU features, for example:
+
+- `supports x86-64-v2`: The CPU supports the x86-64-v2 feature set.
+- `supports x86-64-v3`: The CPU supports the x86-64-v3 feature set.
+
+Supported CPU features (microarchitectures) set are:
+
+- `x86-64`
+- `x86-64-v2`
```

Comment:
Can you search for specific flags or just feature levels?

Reply:
The latter. But it can be, can you think in a good way to call it ? I mean: `supports x86-64-v2` is what I have chosen, how would it look like for a specific flag ?

Reply:
@laverya I have raised this, let me know if it is OK then I will make sure to amend this documentation: https://github.com/replicatedhq/troubleshoot/pull/1631

Reply:
@laverya would you mind doing another review here ?